lib/livebook/intellisense/docs.ex

```diff
@@ -39,9 +39,9 @@ defmodule Livebook.Intellisense.Docs do
   @doc """
   Fetches documentation for the given module if available.
   """
-  @spec get_module_documentation(module()) :: documentation()
-  def get_module_documentation(module) do
-    case Code.fetch_docs(module) do
+  @spec get_module_documentation(module(), node()) :: documentation()
+  def get_module_documentation(module, node) do
```

Comment:
I think it's going to be cleaner if we make it an option :)

```suggestion
  def get_module_documentation(module, opts \\ []) do
```

Reply:
Same with the other functions :)

Reply:
I would go with an argument. Making it a required argument helps us make sure we are always passing it when required. thoughts?

Reply:
Changing the tests is a change we will do once. If we make node an option, then all current code and future code may forget to pass the node, when they should pass it as an argument. I think long term requiring the node will avoid "forgetting", at the cost of a bigger upfront cost. :)